src/signer/noscrypt_cipher.hpp

```diff
@@ -0,0 +1,160 @@
+
+#include <memory>
+
+#include <noscrypt.h>
+#include <noscryptutil.h>
+
+namespace nostr
+{
+    namespace signer
+    {
+        class NoscryptCipherContext
+        {
+        private:
+            NCUtilCipherContext* _cipher;
+
+        public:
+
+			NoscryptCipherContext(uint32_t version, uint32_t mode)
+			{
```

Comment:
The indentation is all over the place here 😆 

I think some lines use tabs and others use spaces.

Stylistically, we can stack namespaces at the same level of indent to keep the indentation on the class and its members more reasonable.

```suggestion
namespace nostr
{
namespace signer
{
class NoscryptCipherContext
{
private:
    NCUtilCipherContext* _cipher;

public:

    NoscryptCipherContext(uint32_t version, uint32_t mode)
    {
```